models/repo.go

```diff
@@ -2611,14 +2636,17 @@ func (repo *Repository) generateRandomAvatar(e Engine) error {
 	}
 
 	repo.Avatar = idToString
-	if err = os.MkdirAll(filepath.Dir(repo.CustomAvatarPath()), os.ModePerm); err != nil {
```

Comment:
So we could remove function `CustomAvatarPath`?

Reply:
@lunny Done